piqueserver/run.py

```diff
@@ -179,20 +188,7 @@ def main():
     if args.json_parameters:
         config.update_from_dict(json.loads(args.json_parameters))
 
-    run = True
-
-    # copy config and update geoip can happen at the same time
-    # note that sys.exit is called from either of these functions on failure
```

Comment:
On failure? Looks like they just always get called to me.

Reply:
hm? that's the removed comment. latest one states:
`# note these functions call sys.exit with codes 0 or 1 based on failure or success`